src/test/kotlin/racingcar/CarsTest.kt

```diff
@@ -0,0 +1,35 @@
+package racingcar
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import racingcar.controller.RandomNumberNumberGenerator
+import racingcar.domain.Cars
+
+class CarsTest {
+    @ParameterizedTest
+    @ValueSource(ints = [4, 5, 6, 7, 8, 9])
+    fun `0에서 9사이의 랜덤 값이 4 이상일 경우 전진한다`(input: Int) {
+        val customRandomNumberNumberGenerator = object : RandomNumberNumberGenerator() {
```

Comment:
```suggestion
        val customRandomNumberNumberGenerator = object : IRandomNumberGenerator() {
```

RandomNumberNumberGenerator 가 아니라 인터페이스를 활용하면 어떤가요?
그렇게 하면 RandomNumberNumberGenerator 가 open class 가 아니라 디폴트 final 클래스로 선언할 수 있을 것 같습니다.